packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -866,6 +868,65 @@ class Transform extends SingleChildRenderObjectWidget {
   }) : transform = new Matrix4.rotationZ(angle),
        super(key: key, child: child);
 
+  /// Creates a widget that transforms its child using a translation.
+  ///
+  /// The `offset` argument must not be null. It specifies the translation.
+  ///
+  /// ## Sample code
+  ///
+  /// This example shifts the silver-colored child down by fifteen pixels.
+  ///
+  /// ```dart
+  /// new Transform.translate(
+  ///   offset: const Offset(0.0, 15.0),
+  ///   child: new Container(
+  ///     padding: const EdgeInsets.all(8.0),
+  ///     color: const Color(0xFF7F7F7F),
+  ///     child: const Text('Quarter'),
+  ///   ),
+  /// )
+  /// ```
+  Transform.translate({
+    Key key,
+    @required Offset offset,
+    this.transformHitTests: true,
+    Widget child,
+  }) : transform = new Matrix4.translationValues(offset.dx, offset.dy, 0.0),
+       origin = null,
+       alignment = null,
+       super(key: key, child: child);
+
+  /// Creates a widget that scales its child uniformly around the center.
+  ///
+  /// The `scale` argument must not be null. It gives the scalar by which
+  /// to multiple the `x` and `y` axes.
```

Comment:
multiple --> multiply coordinates in

Reply:
Fixed the grammar. I've left it as saying the axes are multiplied because I think that's sort of more intuitively what happens? I can change it if you think it's clearer with the "coordinates in" part.